apps/rex/src/app/root-state/toasts/toasts.reducer.ts

```diff
@@ -0,0 +1,18 @@
+import { createReducer, on } from '@ngrx/store';
+import * as ToastStoreActions from './toasts.actions';
+import { initialState } from './toasts.state';
+
+export const toastStoreReducer = createReducer(
+	initialState
+	// on(ToastStoreActions.openSnackBar, state => ({
+	// 	...state,
+	// 	loading: true,
+	// 	error: {},
+	// 	isLoaded: false,
+	// })),
+	// on(ToastStoreActions.closeSnackBar, state => ({
+	// 	...state,
+	// 	isLoaded: false,
+	// 	loading: false,
+	// }))
```

Comment:
clean this up some more and create a follow up tasks for the toast and fixing the appearance of toasts in general